azimuth/plots/dataset_warnings.py

```diff
@@ -575,8 +575,8 @@ def nb_tokens_plot(
     """Create the plot with the dropdown for all labels.
 
     Args:
-        train_per_class_count: Training set count of nb of tokens.
-        eval_per_class_count: Evaluation set count of nb of tokens.
+        train_per_class_count: Training set word count.
+        eval_per_class_count: Evaluation set word count.
```

Comment:
```suggestion
        train_per_class_count: Training set word count per class.
        eval_per_class_count: Evaluation set word count per class.
```
I thought maybe I was just being finicky, but `train_per_class_stats` includes "per class" too... again, lack of "per class" isn't your code, but while you're here, if you don't mind!